src/wp-includes/class-wp-dependencies.php

```diff
@@ -289,6 +289,14 @@ public function add_data( $handle, $key, $value ) {
 		if ( ! isset( $this->registered[ $handle ] ) ) {
 			return false;
 		}
+		if ( 'conditional' === $key ) {
+			_deprecated_argument(
+				'WP_Dependencies->add_data()',
+				'6.9.0',
+				/* translators: 1: WPLANG, 2: wp-config.php */
+				sprintf( __( 'The %1$s argument is no longer supported for inline scripts or styles.' ), 'conditional' )
```

Comment:
The translators comment does not look correct.

---

I've looked at some different deprecated argument usages in Core and there's a mix of things. I wonder whether it's better to describe what's deprecated in a general way vs. literally mentioning the deprecated argument. For example, a message like this describes the change without focusing on the specific argument:

> Conditional scripts and styles are not supported and will not be printed.

Not a leading question, I'm really considering what's best for the user.

Reply:
Huh. I remember correcting that, but apparently I didn't actually do so.

I'm not sure, either, regarding what is most appropriate here. Keeping in mind that "the user" in this case should be a developer.